src/GHC/All.hs

```diff
@@ -131,7 +134,7 @@ ghcFixitiesFromParseFlags = map toFixity . fixities
 
 parseModeToFlags :: ParseFlags -> DynFlags
 parseModeToFlags parseMode =
-    flip lang_set (baseLanguage parseMode) $ foldl' xopt_unset (foldl' xopt_set baseDynFlags enable) disable
+  flip lang_set (baseLanguage parseMode) $ foldl xopt_unset (foldl' xopt_set baseDynFlags enable) disable
```

Comment:
Is the change from `foldl'` to foldl` intentional?

Reply:
no! good catch. thanks!